### PR TITLE
DEX-473: moved initiators to the collaboration model

### PR DESCRIPTION
**Pull request by naknomum**
## Pull Request Overview

- Moves initiators (view, edit) from booleans on the UserAssociations to User guids on the collaboration itself
- _Bonus commit:_ bump the EDM version number higher to correspond with latest code changes there which houston needs

### CODE REVIEW

File: migrations/versions/4663a7a0108b_.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('collaboration', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('initiator_guid', app.extensions.GUID(), nullable=False))
```
Comment (naknomum) [inline migrations/versions/4663a7a0108b_.py:29]: i realize this upgrade is susceptible to same failure that adding non-nullable `email` to `audit_log` had: namely, any existing _Collaborations_ will prevent this schema change from happening, as `initiator_guid` will be null.  ugh, a tougher upgrade.
i guess in a perfect world, i would write some code here to iterate through `collaboration_user_associations`, find the _initiator users_ there, and *set their guid* on `initiator_guid` here.  :disappointed:

Comment (naknomum) [reply migrations/versions/4663a7a0108b_.py]: ok, think i have some hackery in the migration upgrade that does this now.   :crossed_fingers:

**Review by hwindsor — APPROVED** [review]
LGTM, I like the improvements on adding he foreignKey to the guids.

**Pull request merged**